contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -3738,7 +3738,7 @@ freetext_function
 
 freetext_predicate
     : CONTAINS LR_BRACKET (full_column_name | LR_BRACKET full_column_name (COMMA full_column_name)* RR_BRACKET | (table_name DOT)? STAR  | PROPERTY LR_BRACKET full_column_name COMMA expression RR_BRACKET ) COMMA expression RR_BRACKET
-    | FREETEXT LR_BRACKET table_name COMMA (full_column_name | LR_BRACKET full_column_name (COMMA full_column_name)* RR_BRACKET | (table_name DOT)? STAR  ) COMMA expression  (COMMA LANGUAGE expression)? RR_BRACKET
```

Comment:
Why this line is changed ?

Reply:
This is a mistake in the parser rule so we changed it. For example, ... FREETEXT(col_name, 'query') ... is legal, but this rule does not allow it. This change does not impact our implementation of CONTAINS though.